### DISCUSSION TITLE
WebDAV Connectivity Check fails with HTTP 401

### DISCUSSION THREAD

**Discussion by kiranjholla** [remotely-save/remotely-save#249] [Q&A]
I have been trying to setup a WebDAV based sync for Obsidian but my configuration for WebDAV server keeps failing the connectivity check with a HTTP 401 error.

![image](https://user-images.githubusercontent.com/4491818/203587693-c68a8484-6120-422b-bf7d-1af80ad02357.png)

However, I know that the WebDAV server I use works fine. I am actively making use of that WebDAV server for other purposes without incident. I also tried extracting just those pieces of code that seem to run when we click on the *Check* button on the WebDAV connectivity configuration settings page, in the hope of identifying what might be going wrong.

However, that extracted piece of code is also able to successfully connect and retrieve information from my WebDAV server. Hence, at this stage, I am clueless as to why the plugin is failing. My extracted PoC is at [WebDAV Connectivity Check](https://github.com/kiranjholla/webdav-connectivity-check)

My WebDAV server is hosted on Opalstack and uses Digest Authentication. The only thing I can think of, but can't confirm, is that the setting to make it Digest is somehow not getting saved. I say this because that's the only time I was able to reproduce the HTTP 401 error: when I changed the `authType` to `basic` instead of `digest`.

Any pointers on how I can resolve this problem?

**Comment by promexio** [comment]
I have sometimes the same issue. See my bug. Sometimes it works, I only have to try long enough.

 There is nobody providing any help. Very sad :'(

**Comment by fyears** [comment]
redirected to a new issue #445 . i have no clue too but let’s try
